### ISSUE TITLE
train istr with custom dataset

### ISSUE THREAD

**Issue by imenselmi** [hujiecpp/ISTR#16]
how can i train istr with custom dataset ?

**Comment by imenselmi** [comment]
which files should i change?

**Comment by imenselmi** [comment]
also Is it possible to use an ISTR with custom dataset or not ?

**Comment by hujiecpp** [comment]
Yes, please kindly refer to the doc of Detectron2, which introduces how to implement custom dataset to train your own models.
